### PR TITLE
crypto_policy_test.go: Do not hardcode SSP deployment NS

### PR DESCRIPTION
**Pull request by 0xFelix**
**What this PR does / why we need it**:

Use the appropriate getter instead of hardcoding the deployment namespace to 'kubevirt'.

**Release note**:

```release-note
None
```

### CODE REVIEW

File: tests/crypto_policy_test.go [hunk 1]
```diff
@@ -183,10 +183,10 @@ func tryToAccessEndpoint(pod core.Pod, subpath string, port uint16, tlsConfig cl
 		subpath = "/" + subpath
 	}
 
-	// ssp-webhook-service.kubevirt.svc is used to access the endpoints we are testing.
+	// ssp-webhook-service.${deploymentNamespace}.svc is used to access the endpoints we are testing.
 	// It is used here for the metrics as well for simplicity, as it is the CN in the ca_cert
 	// and the metrics just sit on a different port on the same pod.
-	attemptedUrl = fmt.Sprintf("https://ssp-webhook-service.kubevirt.svc:%d%s", port, subpath)
+	attemptedUrl = fmt.Sprintf("https://ssp-webhook-service.%s.svc:%d%s", strategy.GetSSPDeploymentNameSpace(), port, subpath)
```
Comment (dominikholler) [inline tests/crypto_policy_test.go:189]: I am unsure about the expected URL in OpenShift, is it
ssp-operator-service.openshift-cnv.svc
or
ssp-webhook-service.openshift-cnv.svc
?

Comment (0xFelix) [reply tests/crypto_policy_test.go:189]: `ssp-webhook-service` is the only `Service` created by the manifests found in `config/webhook`, so `ssp-webhook-service.${deploymentNamespace}.svc` should be right.

**Comment by lyarwood** [comment]
/lgtm

@ksimon1 do you want to approve this as @akrejcir is out today?

File: tests/crypto_policy_test.go (hunk 1 above)
Comment (akrejcir) [reply tests/crypto_policy_test.go:189]: I think when the webhook is created by OLM, the service may have a different name. But I haven't tried it.

Comment (0xFelix) [reply tests/crypto_policy_test.go:189]: Yes, this is also an issue. I will introduce another getter similar to the NS shortly.

**Pull request merged**